repos/re-theme/src/head/headElement.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component, useEffect, useRef } from 'react'
+import * as ReactDOM from 'react-dom'
+import { Consumer } from './headContext'
+import { hasDomAccess } from '../helpers/hasDomAccess'
+
+const canUseDOM = hasDomAccess()
+
+export const HeadElement = props => {
+
+  const { tag: Tag, style, id, ...tagProps } = props
+  const { tag, name, property } = tagProps
+
+  const indexRef = useRef(-1)
+  const headTagsRef = useRef(null)
+  const children = Tag === 'style' && style || props.children
```

Comment:
I do it that way because the output is not the same when the condition fails
Using a ternary would not give me the result I want when the condition fails
See this example => http://snpy.in/KgePqp

Reply:
I'll update to use `Tag === 'style' && (style || props.children)` style moving forward. I just need to remember to do it. Good catch